Pins-Examples/IT81202B_Pinout_Example_Option1.json

```diff
@@ -0,0 +1,79 @@
+{
+    "partType": "EC",
+    "manufacturer": "ITE",
+    "mpn": "IT81202E",
+    "datasheetVersion": "C0.3.2",
+    "releaseDate": null,
+    "pins": [
+        {
+            "_comment_1_ForWG":" Option1 : Use Arrays",
+            "terminal_identifier": "5",
+            "name": ["ALERT#","SERIRQ","GPM6"],
+            "standardized_name": null,
+            "description": "multiplexed pin",
+            "function":["SPI", "IRQ", "GPIO"],
+            "_comment_2_ForWG": "using GPIO here but the ITE datasheet has specific notations for the GPIO: PIO & EIO",
+            "direction":["inout","in","in"],
+            "polarity": ["low","high","high"],
+            "floatUnused": ["no", "no", "no"],
+            "voltageOptions": {
+                "value":[1.8, 3.3],
+                "unitText": "V",
+                "unitFactor": 1
+            },
+
+            "vihMin":{
+                "_Comment_1_ForWG": "Each voltageOption has a different value. Need to see how to best implement.",
+                "value":[1.35,2],
+                "_Comment_2_ForWG": "Note that for 1.8V, vihMin is listed as 0.75*VSUP",
```

Comment:
Good thing to discuss - generally a percentage of VDD - should we add this to the unit/value definition?